sfdb.py

```diff
@@ -370,7 +370,7 @@ def scanLogEvent(self, instanceId, classification, message, component=None):
             if "locked" in e.args[0] or "thread" in e.args[0]:
                 # TODO: Do something smarter here to handle locked databases
                 #print("[warning] Couldn't log due to SQLite limitations. You can probably ignore this.")
-                #self.sf.fatal("Unable to log event in DB due to lock: " + e.args[0])
+                self.sf.debug("Unable to log event in DB due to lock: " + e.args[0])
```

Comment:
Actually it needs to be print because it gets here in the first place from `self.sf.debug`.

Reply:
Is there a reasonable no-op we can use here instead in order to preserve the code format?